### PR TITLE
github: add @RanderWang as owner of haswell

### PR DESCRIPTION
**Pull request by RanderWang**
I would like to review any change in haswell domain

Signed-off-by: Rander Wang <rander.wang@linux.intel.com>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -26,10 +26,14 @@ src/audio/tone.c			@singalsu
 src/arch/xtensa/*			@tlauda
 src/platform/*				@tlauda
 src/platform/baytrail/*			@xiulipan
-src/platform/haswell/*			@xiulipan
+src/platform/haswell/*			@xiulipan @randerwang
 
 # drivers
 src/drivers/intel/cavs/dmic.c		@singalsu
+src/drivers/intel/cavs/hda.c		@randerwang
```
Comment (jajanusz) [inline CODEOWNERS]: I think you don't want to do this. It makes you one and only codeowner of hda.
It's not just a watchlist. You can watch only on repo level by using github built-in features (watch on the top).
If you want to get notifications on directory/file-level you can try something like in this answer: https://stackoverflow.com/a/25315476

Comment (RanderWang) [reply CODEOWNERS]: sorry, I don't make it clear in my commit message. I want to review change in some files.
I will refine my file list.

**Comment by RanderWang** [comment]
update my PR, thanks!

**Comment by lgirdwood** [comment]
@zrombel This change is not part of build so I'm not sure why CI build is failing here ?

**Comment by zrombel** [comment]
The PR was rebuilt successfully, seems like the notification didn't reach github. I will take a look at it nad report QB error if needed.

**Pull request merged**